dask/dataframe/tests/test_dataframe.py

```diff
@@ -5108,6 +5108,19 @@ def test_index_is_monotonic_dt64():
     assert_eq(s_2.index.is_monotonic_decreasing, ds_2.index.is_monotonic_decreasing)
 
 
+def test_is_monotonic_empty_partitions():
+    df = pd.DataFrame({"a": [1, 2, 3, 4], "b": [4, 3, 2, 1]})
+    ddf = dd.from_pandas(df, npartitions=2)
+
+    # slice it to get empty partitions
+    df = df[df["a"] >= 3]
+    ddf = ddf[ddf["a"] >= 3]
+    assert_eq(df["a"].is_monotonic, ddf["a"].is_monotonic)
+    assert_eq(df.index.is_monotonic, ddf.index.is_monotonic)
```

Comment:
Apologies for not catching this before, but we should use `is_monotonic_increasing` since `is_monotonic` is deprecated. In fact, when I run this test locally I get a failure due to the deprecation warning being elevated to an error (which is the behavior I'm expecting). I'm not sure why we're not seeing the same thing in CI 

```suggestion
    assert_eq(df["a"].is_monotonic_increasing, ddf["a"].is_monotonic_increasing)
    assert_eq(df.index.is_monotonic_increasing, ddf.index.is_monotonic_increasing)
```

Reply:
Ah, it's because this deprecation came to my attention from the `upstream` build where the `pandas` release with this deprecation hasn't been released yet (xref https://github.com/dask/dask/pull/8653). So that explains why we're not seeing it in CI here, but it's still worth changing as this test will fail in our `upstream` build without the `is_monotonic` -> `is_monotonic_increasing` change

Reply:
ah yep! I had a nagging sense that something was deprecated about is_monotonic but I couldn't quite remember what it was.